modules/library/api/src/main/java/org/geotools/geometry/jts/ReferencedEnvelope.java

```diff
@@ -875,21 +897,7 @@ public static ReferencedEnvelope reference(Envelope e) {
             return new ReferencedEnvelope(e, null);
         }
     }
-
```

Comment:
method is a duplicate to reference(org.opengis.geometry.Envelope env), which is not needed